wezterm-gui/src/termwindow/mod.rs

```diff
@@ -3460,6 +3462,11 @@ impl TermWindow {
                     tab_index: idx,
                     tab_id: tab.tab_id(),
                     is_active: tab_index == idx,
+                    is_last: if let Some(tab_index_last) = window.get_last_active_idx() {
+                        tab_index_last == idx
+                    } else {
+                        false
+                    },
```

Comment:
I think we can make this a bit more succinct:

```suggestion
                    is_last: window.get_last_active_idx().map(|last| last == idx).unwrap_or(false),
```